shipit-engine.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.rdoc"]
   s.test_files = Dir["test/**/*"]
 
-  s.add_dependency 'rails', '~> 4.2.2'
+  s.add_dependency 'rails', '~> 4.2.0'
```

Comment:
> If it is worth noting, 4.2.1 is vulnerable to some security issues

Yeah. I adjusted my `Gemfile` to use the pessimistic operator for `4.2.x` and ran `bundle update 'rails'` update my `Gemfile.lock` to get Rails 4.2.3.

> Yeah, definitely not this gem job to make sure people don't use vulnerable
versions :+1:

I agree